server/node.go

```diff
@@ -585,97 +585,97 @@ func (n *nodeServer) executeCmd(args proto.Request, reply proto.Response) error
 
 // TODO(spencer): fill in method comments below.
 
-// Get .
+// Get.
```

Comment:
Per my other comment, let's nix these comments altogether unless golint is requiring them. Pretty sure golint isn't because when golint requires a documentation comment there needs to be a space between the method name and the period.